File: www/src/Lib/_warnings.py
"""_warnings provides basic warning filtering support.
It is a helper module to speed up interpreter start-up."""


default_action = """default"""

filters = [('ignore', None, DeprecationWarning, None, 0), 
    ('ignore', None, PendingDeprecationWarning, None, 0), 
    ('ignore', None, ImportWarning, None, 0), 
    ('ignore', None, BytesWarning, None, 0)]

once_registry = {}

def warn(*args,**kw):
    """Issue a warning, or maybe ignore it or raise an exception."""
    pass

def warn_explicit(*args,**kw):
    """Low-level inferface to warnings functionality."""
    pass

File: www/src/libs/_warnings.js
var $module = (function($B){

_b_ = $B.builtins

return  {
    __doc__: "_warnings provides basic warning filtering support.\n " +
        "It is a helper module to speed up interpreter start-up.",

    default_action: "default",

    filters: [
        ['ignore', _b_.None, _b_.DeprecationWarning, _b_.None, 0],
        ['ignore', _b_.None, _b_.PendingDeprecationWarning, _b_.None, 0],
        ['ignore', _b_.None, _b_.ImportWarning, _b_.None, 0],
        ['ignore', _b_.None, _b_.BytesWarning, _b_.None, 0]].map(
            function(x){return _b_.tuple.$factory(x)}),

    once_registry: _b_.dict.$factory(),

    warn: function(){},

    warn_explicit: function(){}

}

})(__BRYTHON__)
